bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -141,60 +177,87 @@ final class BagsManager(executorService: ExecutorService) {
 
         val key = obs.getNodeKey
         state += key
+        if (bagsStatus(key).isEmpty)
+          setStatus(key, BagsStatus.NewPending.some)
+
         Future {
+          // Wait the delay before running.
+          Thread.sleep(delay)
```

Comment:
If, say, a catalog lookup fails due to lack of internet connectivity or for some other reason, we don't want the request to be immediately re-enqueued and then continue to hammer away and fail, so we want a delay before we re-execute and allow the net connection to come back up, the catalog server to recover, etc.